scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/Readme.md

```diff
@@ -0,0 +1,172 @@
+# IPC Footprint Generator for No Lead Packages
+
+This generator uses IPC-7351B equations and fillet definitions to derive a footprint for no-lead style packages from the dimensions of the package. (The suggested footprint is only used to determine the size of the exposed pad.) Fillet size definitions can be changed by pointing to a personalized IPC parameter file. The script generates footprints in zero orientation version A (pin 1 at top left corner).
```

Comment:
The suggested footprint is only optionally used to determine the EP size, right? This sentence sounds like it's mandatory but the EP size can be calculated from package info and also ignore the suggested footprint.

Reply:
Fixed also in gullwing readme